library/odoo

```diff
@@ -1,6 +1,6 @@
 Maintainers: Christophe Monniez <moc@odoo.com> (@d-fence)
 GitRepo: https://github.com/odoo/docker
-GitCommit: 0a794bf1c202f1d7ca409ce30c159350dc541100
+GitCommit: b4cccf3ccae004aebfd334aa2639499496151598
 
 Tags: 17.0-20240730, 17.0, 17, latest
```

Comment:
You probably want to update all these date references too (to `20240809`) :eyes:

(The way we mitigate this oversight for repos we maintain is by writing a script that generates this `library/XXX` content directly from our repo metadata, in case that's an idea that would be interesting/helpful for you :eyes:)

Reply:
Nice catch, thanks.